src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
```

Comment:
If you call the hide method again your animal will unhide. Are you sure it is right?

Reply:
yes, you're right. Animal starts to be unhide. It's not normal. Help me, please, to implement this case using point #3 of our checklist https://github.com/mate-academy/js_herbivores_and_carnivores/blob/master/checklist.md. Thank you!

Reply:
is it correct?
  hide() {
    if (this.hidden) {
      this.hidden = this.hidden;
    }
    this.hidden = !this.hidden;
  }